spec/features/user_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
```

Comment:
- [ ] When I run tests in your features folder, I run into a number of errors like in the screenshot below. This is because Chrome Browser cannot be found. f you are not running into these errors, let the previous reviewer know by adding screenshots. One way to solve this issue is to use `type: :request` instead of `type: :system`
![image](https://user-images.githubusercontent.com/45460192/224178665-440c0a48-911e-4698-a4d5-8eb4cf3a5f50.png)

Reply:
Fixed.